barycorrpy/barycorrpy.py

```diff
@@ -19,11 +18,22 @@
 ### Need to install jplephem ###
 #de430 is 100 MB in size
 
-def exposure_meter_BC_vel(JDUTC,expmeterflux,
-       hip_id=0, ra=0., dec=0., epoch=2451545., pmra=0., pmdec=0., px=0.,
-       obsname='', lat=0., longi=0., alt=0., rv=0., zmeas=0.,
-       ephemeris='de430', leap_dir=os.path.join(os.path.dirname(__file__),'data'), leap_update=True):
-            
+# Parsing constants #
+AU = const.astronomical_unit # [m]
+c = const.c # Speed of light [m/s]
+pctoau = 3600*180/np.pi # No. of AU in one parsec
+year = 365.25*3600*24 # [s]
+kmstoauyr = year/(1000*AU)
+
+M_moon = 73476730924573500000000 # Mass of the Moon [kg]
+
+# Mass and normalised mass of solar system bodies
+ss_bodies = ['Sun', 'Mercury', 'Venus', 'Earth', 'Mars', 'Jupiter', 'Saturn', 'Uranus', 'Neptune', 'Moon']
+M = dict(zip(ss_bodies, [u.M_sun.value, 0.3301e24, 4.867e24, u.M_earth.value, 0.6417e24, u.M_jup.value, 568.5e24, 86.82e24, 102.4e24, M_moon])) # [kg]
+GM = {k:const.G*M[k] for k in ss_bodies}
+
+
+def exposure_meter_BC_vel(JDUTC, expmeterflux, **kwargs):
```

Comment:
Hi,
Would like to leave the list of parameters here, even though it looks bulky since it allows the user to see the input needed to run the code. Don't want to leave this as a variable number since the default value of everything is set to 0, which could potentially hamper the precision of the correciton

Reply:
As you like.
* But I think it makes this wrapper easier to maintain, when adding for instance new keywords to `get_BC_vel`. Also it is directly to see that the other keywords are the same (as done btw in the docstring).
* The default values for kwargs in `exposure_meter_BC_vel` will be the same as `get_BC_vel` (0, 0., None, ephemeris, leap_dir, etc.). Another thing that does not need to be double checked!
* The number of arguments in `exposure_meter_BC_vel` is not really variable, since  `get_BC_vel` will complain if  `exposure_meter_BC_vel` is called with additional keywords.

So no risk to hamper the precision.
Moreover, I would move `exposure_meter_BC_vel` to the end of the file.

Reply:
Hi,
We considered that. 
1. We cannot add new keywords to get_BC_vel now because existing users of the code will be thrown an error. 
2. By number of parameters, I meant more that the user may enter too few and forget about about a few. 

Sure, would you like to do this (undo kwargs and then  moving it to the end of the file ) by editing this request, or should I accept the request and make the changes?

Reply:
*undo kwargs and move the exposure meter function to the end.

Reply:
Please modify my request as you like.